tf2_kdl/test/test_tf2_kdl.cpp

```diff
@@ -109,23 +107,23 @@ TEST(TfKDL, ConvertVector)
 
 int main(int argc, char **argv){
   testing::InitGoogleTest(&argc, argv);
-  ros::init(argc, argv, "test");
-  ros::NodeHandle n;
 
-  tf_buffer = new tf2_ros::Buffer();
+  rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_SYSTEM_TIME);
+  tf_buffer = std::make_unique<tf2_ros::Buffer>(clock);
 
   // populate buffer
-  geometry_msgs::TransformStamped t;
+  geometry_msgs::msg::TransformStamped t;
   t.transform.translation.x = 10;
   t.transform.translation.y = 20;
   t.transform.translation.z = 30;
   t.transform.rotation.x = 1;
-  t.header.stamp = builtin_interfaces::msg::Time(2.0);
+  t.header.stamp.sec = 2.0;
+  t.header.stamp.nanosec = 0.0;
   t.header.frame_id = "A";
   t.child_frame_id = "B";
   tf_buffer->setTransform(t, "test");
 
   int retval = RUN_ALL_TESTS();
-  delete tf_buffer;
+  tf_buffer.reset();
```

Comment:
With the change to a `unique_ptr`, this shouldn't be necessary; it will be destroyed as it goes out of scope.  So you can just remove this.

Reply:
This is still unresolved in the latest patches.

Reply:
Done